packages/veritone-widgets/src/redux/modules/mediaDetails/saga.js

```diff
@@ -637,40 +638,39 @@ function* fetchEntities(widgetId, entityIds) {
           coverImageUrl
         }
       }
-    `;
-  });
+    }
+  }`;
 
   const config = yield select(configModule.getConfig);
   const { apiRoot, graphQLEndpoint } = config;
   const graphQLUrl = `${apiRoot}/${graphQLEndpoint}`;
   const token = yield select(authModule.selectSessionToken);
+  const variables = { entityIds };
 
   let response;
   try {
     response = yield call(fetchGraphQLApi, {
       endpoint: graphQLUrl,
-      query: `query{${entityQueries.join(' ')}}`,
-      token
+      query: entitiesQuery,
+      token,
+      variables
     });
   } catch (error) {
     return yield put({
       type: REQUEST_ENTITIES_FAILURE,
-      error: 'Error fetching entities from server.',
-      meta: { widgetId }
+      error: 'Error fetching entities from server.'
     });
   }
 
   if (response.errors) {
     return yield put({
       type: REQUEST_ENTITIES_FAILURE,
-      error: 'Error thrown while fetching entities',
-      meta: { widgetId }
+      error: 'Error fetching entities from server.'
     });
   }
   yield put({
     type: REQUEST_ENTITIES_SUCCESS,
-    payload: response.data,
-    meta: { widgetId }
+    payload: response.data.entities.records
```

Comment:
Should we use a get here? I don't think it will get here because of the return when there is an error in the response but just in case?